src/utils/update/mapper.utils.ts

```diff
@@ -125,7 +125,7 @@ export const lowerCaseAllWordsExceptFirstLetters = (country: string | undefined)
   const wordsForAllLowerCase = ["AND", "OF", "THE", "DA", "PART"];
 
   return country.replace(/\w*/g, word => {
-    if (word === "MCDONALD") {
+    if (word === "MCDONALD" || word === "Mcdonald") {
```

Comment:
if (word.toUpperCase() == "MCDONALD")
would also do it

Reply:
yeah added its as its easier to understand what's being ignored